db/schema.rb

```diff
@@ -95,9 +95,8 @@
   create_table "budget_ballots", force: :cascade do |t|
     t.integer  "user_id"
     t.integer  "budget_id"
-    t.datetime "created_at",                     null: false
-    t.datetime "updated_at",                     null: false
-    t.integer  "ballot_lines_count", default: 0
```

Comment:
This is 👍 , I messed up somehow on a PR and didn't realize it. I added this on a feature branch that is still on PR stage https://github.com/consul/consul/pull/1712/files#diff-ddcb0cf5e4dfc6ac0c06b85bac771760 My apologies >_< 🙇